guardrails/cli/configure.py

```diff
@@ -59,6 +59,18 @@ def configure(
         help="Opt out of anonymous metrics collection.",
         prompt="Enable anonymous metrics reporting?",
     ),
+    token: Optional[str] = typer.Option(
+        None,
+        "--token",
+        help="API Key for Guardrails. If not provided, you will be prompted for it.",
+    ),
+    remote_inferencing: Optional[bool] = typer.Option(
+        DEFAULT_USE_REMOTE_INFERENCING,
```

Comment:
"remote inferencing" sounds funny. Fine to switch to "remote inference" or "remote inferring" everywhere?